pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -73,8 +74,9 @@ func New(ctx provider) (*Operation, error) {
 		client:  didExchange,
 		service: didexchangeSvc,
 		// TODO channel size - https://github.com/hyperledger/aries-framework-go/issues/246
-		actionCh: make(chan service.DIDCommAction, 10),
-		msgCh:    make(chan service.StateMsg, 10),
+		actionCh:        make(chan service.DIDCommAction, 10),
+		msgCh:           make(chan service.StateMsg, 10),
+		webhookNotifier: webhook.Notifier{SubscriberURLs: webhookURLs},
```

Comment:
Better to use consistent name - SubscriberURLs v webhookURLs.

Reply:
Done